src/handler/grpc/request/search/mod.rs

```diff
@@ -119,6 +125,35 @@ impl Default for Searcher {
 
 #[tonic::async_trait]
 impl Search for Searcher {
+    async fn grpc_search(
```

Comment:
Can we rename it to `search`, because it is in grpc, don't need to add a `grpc` prefix again.

Reply:
renamed